bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/directentrylogger/DirectWriter.java

```diff
@@ -49,7 +49,6 @@ class DirectWriter implements LogWriter {
     final List<Future<?>> outstandingWrites = new ArrayList<Future<?>>();
     Buffer nativeBuffer;
     long offset;
-    private volatile boolean useFallocate = true;
```

Comment:
It's better to use `static`

Reply:
It makes sense to me. I have addressed your comment. PTAL.